modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/styles/components/_activation-keys.scss

```diff
@@ -0,0 +1,18 @@
+.cp-activation-keys-drop-down-item:hover {
+	background-color: var(--color-action-primary-hover-lighten);
+	color: var(--color-action-primary-hover) !important;
+
+	svg {
+		color: var(--color-action-primary-hover) !important;
```

Comment:
These !importants are necessary?

Reply:
Yes, without them, the color won't be apply.